### PR TITLE
Optimized the Switchboard code added error handling 

### PR DESCRIPTION
**Pull request by vishalsinghji**

### CODE REVIEW

**Comment by vishalsinghji** [comment]
@danicuki @brianlong @steveluscher  Can anyone review this PR?

File: code/switchboard/client/read.client.en.ts
Comment (steveluscher) [inline code/switchboard/client/read.client.en.ts:17]: You can fetch these in parallel with `Promise.all()`. This will reduce the latency to the slower of the two, rather than the sum of both.

File: code/switchboard/client/read.client.en.ts
Comment (steveluscher) [inline code/switchboard/client/read.client.en.ts:26]:
```suggestion
    const result = await aggregatorAccount.getLatestValue();
    console.log("Latest value:", result.toNumber());
```

File: code/switchboard/client/read.client.en.ts
Comment (steveluscher) [inline code/switchboard/client/read.client.en.ts:25]: You removed the `any` annotation here. If that was intentional then can you also update `read.client.preview.en.ts`?

**Review by steveluscher — CHANGES_REQUESTED** [review]
Thanks for this!

It took me way too long to find where this code is on the Cookbook. Adding a direct link to the content in your PR description to make it easier on your reviewers might get you a review faster!

File: code/switchboard/client/read.client.en.ts
Comment (vishalsinghji) [reply code/switchboard/client/read.client.en.ts]: I have used Promise.all() to reduce the latency.

**Review by steveluscher — APPROVED** [review]
I don't have merge rights to this repo, but LGTM.

**Review by jacobcreech — APPROVED** [review]

**Pull request merged**
